### PR TITLE
[FIX] general : Update link from Google Domains to Squarespace

### PR DESCRIPTION
**Pull request by masp-odoo**
Task-4224635

### CODE REVIEW

File: content/applications/general/email_communication/email_domain.rst [hunk 1]
```diff
@@ -141,9 +141,14 @@ SPF, DKIM and DMARC documentation of common providers
...
 - `CloudFlare DNS <https://support.cloudflare.com/hc/en-us/articles/360019093151>`_
-- `Google Domains <https://support.google.com/domains/answer/3290350?hl=en>`_
+- `Squarespace DMARC record <https://support.squarespace.com/hc/en-us/articles/360002101888-Adding-custom-DNS-records-to-your-Squarespace-managed-domain#toc-dmarc-records>`_
+- `Squarespace DKIM & SPF records <https://support.squarespace.com/hc/en-us/articles/360002101888-Adding-custom-DNS-records-to-your-Squarespace-managed-domain#toc-dkim-and-spf-records>`_
 - `Azure DNS <https://docs.microsoft.com/en-us/azure/dns/dns-getstarted-portal>`_
 
+.. note::
+   In September 2023, Google sold its domain registrar business, Google Domains, to Squarespace.
+   From now on, Squarespace handles new domain registrations under its own platform.
+
```
Comment (xpl-odoo) [inline content/applications/general/email_communication/email_domain.rst:151]:
```suggestion
```
Not necessary IMO

File: content/applications/general/email_communication/email_domain.rst [hunk 2]
```diff
@@ -141,9 +141,14 @@ SPF, DKIM and DMARC documentation of common providers
 - `GoDaddy CNAME record <https://www.godaddy.com/help/add-a-cname-record-19236>`_
 - `NameCheap <https://www.namecheap.com/support/knowledgebase/article.aspx/317/2237/how-do-i-add-txtspfdkimdmarc-records-for-my-domain/>`_
 - `CloudFlare DNS <https://support.cloudflare.com/hc/en-us/articles/360019093151>`_
-- `Google Domains <https://support.google.com/domains/answer/3290350?hl=en>`_
+- `Squarespace DMARC record <https://support.squarespace.com/hc/en-us/articles/360002101888-Adding-custom-DNS-records-to-your-Squarespace-managed-domain#toc-dmarc-records>`_
+- `Squarespace DKIM & SPF records <https://support.squarespace.com/hc/en-us/articles/360002101888-Adding-custom-DNS-records-to-your-Squarespace-managed-domain#toc-dkim-and-spf-records>`_
```
Comment (xpl-odoo) [inline content/applications/general/email_communication/email_domain.rst:145]:
```suggestion
- `Squarespace DNS records <https://support.squarespace.com/hc/en-us/articles/360002101888-Adding-custom-DNS-records-to-your-Squarespace-managed-domain>`_
```
Just a single link to the main page should be sufficient.

**Review by xpl-odoo — APPROVED** [review]
It's alright like this, so I'm already delegating the r+, but I would make the following changes :)
@robodoo delegate+

**Comment by masp-odoo** [comment]
Alright, @xpl-odoo, I made the changes (actually, what you proposed is what I originally wrote but changed to add a bit of context). Thank you for your review!
@robodoo r+

**Pull request closed without merging**